src/components/Table/Paginator.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Flex,
+  IconButton,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+  Select,
+  Text,
+  Tooltip,
+} from '@chakra-ui/react';
+import { ArrowLeftIcon, ArrowRightIcon, ChevronLeftIcon, ChevronRightIcon } from '@chakra-ui/icons';
+import { TableInstance } from 'react-table';
+import { BaseObject } from 'model/base-object';
+
+function Paginator<T extends BaseObject>(props: TableInstance<T>) {
+  return (
+    <Flex
+      justifyContent="space-between"
+      alignItems="center"
+      m={{ base: 0, lg: 4 }}
+      mt={{ base: 4 }}
+      flexWrap={{ base: 'wrap', lg: 'nowrap' }}
+      flexDirection={{ base: 'row' }}
+    >
+      <Flex order={1}>
+        <Tooltip label="Pierwsza strona">
+          <IconButton
+            aria-label="Pierwsza strona"
+            onClick={() => props.gotoPage(0)}
```

Comment:
Destructure the props as much as possible everywhere please.

Reply:
We came to the conclusion (at the meeting on 23.04.2021) that destructing isn't required and necessary due to code duplication (types are declared near `props` in those components).